### PR TITLE
Add missing semi-colons

### PR DESCRIPTION
**Pull request by wilsonge**
This adds some semi colons missing from the file. Note that this has been applied automatically to the compressed file so there are no issues there

### CODE REVIEW

File: media/system/js/core-uncompressed.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ Joomla.renderMessages = function(messages) {
 				var messageWrapper = document.createElement('p');
 				messageWrapper.innerHTML = typeMessages[i];
 				messagesBox.appendChild(messageWrapper);
-			};
+			}
```
Comment (wilsonge) [inline media/system/js/core-uncompressed.js]: This was at the end of a for() statement so was not needed

**Comment by nonumber** [comment]
Maybe also add the missing semi-colons to lines 151, 230 & 231

**Comment by N6REJ** [comment]
lol, how in the world did this pass travis?
Bear

**Comment by wilsonge** [comment]
Travis only tests the things we have unit tests for. Which is less than 50% of our php code in the libraries folder. Nothing else is covered by travis

**Comment by nueckman** [comment]
Hi George, please fix line 230 & 231 too.

**Comment by RoterNagel** [comment]
@nueckman Line 231, 232 ?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6106">issues.joomla.org/joomla-cms/6106</a>.</sub>

**Comment by nonumber** [comment]
231, 232: Add semi-colon to end.

**Comment by wilsonge** [comment]
My bad - done

**Comment by nonumber** [comment]
@test success<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6106">issues.joomla.org/joomla-cms/6106</a>.</sub>

**Comment by Kubik-Rubik** [comment]
Tested successfully, thank you @wilsonge!

**Pull request merged**